run-tests.php

```diff
@@ -1350,20 +1350,20 @@ function run_test($php, $file, $env)
 
 		} else {
 
-			if (!isset($section_text['PHPDBG']) && @count($section_text['FILE']) + @count($section_text['FILEEOF']) + @count($section_text['FILE_EXTERNAL']) != 1) {
-				$bork_info = "missing section --FILE--";
+			if (!isset($section_text['PHPDBG']) && !isset($section_text['FILE']) && !isset($section_text['FILEEOF']) && !isset($section_text['FILE_EXTERNAL'])) {
```

Comment:
Formerly, the condition caught tests with multiple FILE* sections also (albeit with a misleading error message). Is that taken into account elsewhere?

Reply:
AFAIK, only duplicated sections are detected:

https://github.com/php/php-src/blob/bff179888f6d247c3584150f0c729634b95c745b/run-tests.php#L1304

Maybe duplicate `FILE`* and `EXPECT*` sections should be reported as well?

Reply:
Well, as it is now (i.e. before the patch), duplicate `FILE*` and `EXPECT*` sections are reported (with a bogus error message, though), so this behavior should be retained. Maybe we could use something like
````php
if (count(array_intersect_key($section_text, array_flip(['EXPECT', 'EXPECTF', 'EXPECTREGEX']))) != 1) {
    $bork_info = "either of --EXPECT--, --EXPECTF-- or --EXPECTREGEX-- is required";
}
````

Reply:
I have tested, and duplicated `FILE*` and `EXPECT*` aren't reported if there aren't the same. But let's add it, right? :smile:

Reply:
@cmb69 I change the condition to `> 1`, as some tests, i.e. `PHPDBG`, only have the `--PHPDBG--` section.